e2e/wdio/core/tests/select.e2e-spec.ts

```diff
@@ -0,0 +1,398 @@
+            expect(textBefore).not.toEqual(textAfter);
+        });
+
+        it('should have visual check', () => {
+            click(mobileModeExample + buttons, 0);
+            waitForElDisplayed(option);
+            const arrLength = getElementArrayLength(option);
+
+            for (let i = 0; arrLength > i; i++) {
+                checkElementStates(option, mobileSelectOptions + i, optionTag, i);
+            }
+        });
+    });
+
+    describe('Max Height', function() {
+        it('should be able to select the option', () => {
+            const textBefore = getText(maxHeightExample + displayedText);
+            click(maxHeightExample + buttons, 0);
+            waitForElDisplayed(option, 4);
+            click(options(47));
+            const textAfter = getText(maxHeightExample + displayedText);
+            expect(textBefore).not.toEqual(textAfter);
+        });
+
+        it('should have visual check', () => {
+            click(maxHeightExample + buttons, 0);
+            waitForElDisplayed(option, 4);
+            const arrLength = getElementArrayLength(option);
+
+            for (let i = 4; arrLength > i; i++) {
+                checkElementStates(option, maxHeightSelectOptions + i, optionTag, i);
+            }
+        });
+    });
+
+    describe('Adding and Removing Options', function() {
+        beforeEach(() => {
+            refreshPage();
+        });
+
+        it('should be able to select the option', () => {
+            const textBefore = getText(addRemoveOptionExample + displayedText);
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            click(options(56));
+            const textAfter = getText(addRemoveOptionExample + displayedText);
+            expect(textBefore).not.toEqual(textAfter);
+        });
+
+        it('should be able to add option', () => {
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const optionsCountBefore = getElementArrayLength(option);
+            click(addRemoveOptionExample + buttons, 2);
+            click(addRemoveOptionExample + buttons, 0);
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const optionsCountAfterAdding = getElementArrayLength(option);
+
+            expect(optionsCountBefore).toEqual(optionsCountAfterAdding - 1);
+        });
+
+        it('should be able to add remove option', () => {
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const optionsCountBefore = getElementArrayLength(option);
+            click(addRemoveOptionExample + buttons, 2);
+            click(addRemoveOptionExample + buttons, 1);
+            click(addRemoveOptionExample + buttons, 2);
+            waitForElDisplayed(option, 4);
+            const optionsCountAfterAdding = getElementArrayLength(option);
```

Comment:
if removing, then please rename to optionsCountAfterRemoving

Reply:
Fixed